src/handlers/payout/action.ts

```diff
@@ -5,6 +5,8 @@ import { Payload, StateReason } from "../../types";
 import { shortenEthAddress } from "../../utils";
 import { bountyInfo } from "../wildcard";
 
+const TEMPLATE_PAYOUT_URL = "https://pay.ubq.fi?claim";
```

Comment:
Makes sense but I fear that it's possible the link is part of a comment for any reason. 

I think it's critical for debugging purposes to log a descriptive comment explaining why no permit was generated if this is the reason.

Reply:
I think maybe mentioning the comment which does posses the URL makes sense, let's say I added the URL in one of comments before generating the permit but due the comment permit won't be generated mentioning the comment maybe PR reviewer can delete that comment and re-generate the permit?

Reply:
If not we should log every permit generation into supabase with the associated Issue and User.